docs/build/js-client/3-setting-up.md

```diff
@@ -1,27 +1,49 @@
 # Setting up project
 
-## CLI install
+### Intro
+
+The instruction below will guide you through the process of setting up an environment for JS client.
+
+> Created structure could look intimidating for a simple project, but its battle tested and extensible if the need arises. 
+
+### Setting up for frontend
 
 The best way to initialize the project is by using CLI
 
 > You can find an installation guide [here](https://github.com/fluencelabs/cli).
```

Comment:
```suggestion
> You can find an installation guide for the [Fluence CLI](https://github.com/fluencelabs/cli).
```
that's probably the better url: https://github.com/fluencelabs/cli?tab=readme-ov-file#installation-and-usage